cli/src/bin/phylum.rs

```diff
@@ -190,29 +150,58 @@ async fn handle_commands() -> CommandResult {
         )
     })?;
 
-    if matches.subcommand_matches("ping").is_some() {
-        let resp = api.ping().await;
-        print_response(&resp, true, None);
-        return Ok(ExitCode::Ok.into());
-    }
+    let api = async {
+        PhylumApi::new(
+            &mut config.auth_info,
+            &config.connection.uri,
+            timeout,
+            ignore_certs,
+        )
+        .await
+        .context("Error creating client")
+    };
```

Comment:
`async` blocks are weird... I fear this makes the code a bit too hard to read.

Reply:
I've thought a bit about making more use out of the `config` object so that we can pass that around instead of passing `ignore_certs`, `api`, and `timeout` all separately. But that can wait for a future PR.